.redhat-ci.Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN dnf install -y \
         gjs \
         parallel \
         clang \
-        libubsan \
+        lib{ub,a,t}san \
```

Comment:
This isn't going to take effect until it's part of the tree, so you'll also have to explicitly list the new packages in the YAML until then. I suppose that's probably why the `cc` sanity check is failing.